src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -1266,9 +1248,6 @@ public static Object get(final Object object, final int index) {
         } else if (object instanceof Iterable<?>) {
             final Iterable<?> iterable = (Iterable<?>) object;
             return IterableUtils.get(iterable, i);
-        } else if (object instanceof Collection<?>) {
```

Comment:
Is this one really safe to remove? It seems to assume that the else condition will handle with the same behaviour, but I have not tested it to confirm.

Might be worth also moving that `if (object == null)` check to the top.